modules/dnn/src/net_quantization.cpp

```diff
@@ -32,8 +32,13 @@ void getQuantizationParams(const Mat& src, std::vector<float>& scales, std::vect
     zeropoints.push_back((int)std::round(zp));
 }
 
-// FIXIT drop from inference API
 Net Net::Impl::quantize(InputArrayOfArrays calibData, int inputsDtype, int outputsDtype)
+{
+    return quantize(calibData, inputsDtype, outputsDtype, false);
+}
```

Comment:
Do we need this method? (in internal class)

Reply:
Thanks for code reviewing. 
How about removing `Net Net::Impl::quantize(InputArrayOfArrays calibData, int inputsDtype, int outputsDtype)` and just keeping `Net Net::Impl::quantize(InputArrayOfArrays calibData, int inputsDtype, int outputsDtype, bool perTensor = false)`?

Reply:
We could remove this if it is not used anymore.